xds/src/main/java/io/grpc/xds/internal/sds/SdsProtocolNegotiators.java

```diff
@@ -264,12 +266,24 @@ public void exceptionCaught(ChannelHandlerContext ctx, Throwable cause)
         localXdsClientWrapperForServerSds =
             XdsClientWrapperForServerSds.newInstance(port, Bootstrapper.getInstance(), syncContext);
       } catch (Exception e) {
-        logger.log(Level.WARNING, "Exception while creating the xDS client", e);
+        logger.log(Level.FINE, "Fallback to plaintext due to exception", e);
         localXdsClientWrapperForServerSds = null;
       }
       this.xdsClientWrapperForServerSds = localXdsClientWrapperForServerSds;
     }
 
+    @VisibleForTesting
+    public ServerSdsProtocolNegotiator(
```

Comment:
nit: it would be nice if the other constructor is calling this one (or could be a factory method).